buf.yaml

```diff
@@ -0,0 +1,28 @@
+# buf.yaml files should always be in the root of your repository.
+build:
+  roots:
+    # roots is the directories that all .proto files are contained within, and
+    # all imports must derive from.
+    # Only .proto files within these directory will be compiled.
+    - model
+  # excludes is the list of directories within root to exclude.
+  excludes:
+    - model/build
+
+# lint contains the options for lint checks.
+lint:
+  # use is the list of checker categories and ids to use for buf check lint.
```

Comment:
Nit: start comments with capital for consistency.

Ditto elsewhere.

Reply:
Done

Reply:
This has been done here and elsewhere in this file but I cannot resolve this as @BenHenning  is the reviewer and he will need to resolve this.